guacamole/src/main/frontend/src/app/home/styles/home.css

```diff
@@ -79,4 +80,21 @@ a.home-connection, .empty.balancer a.home-connection-group {
 .header-app-name {
     font-size: 0.85em;
     box-shadow: none;
+}
+
+.recent-connections .connection .x-thumbnail {
```

Comment:
`x-thumbnail` doesn't seem quite right for a CSS class name - it's describing what the element looks like, not what it _is_.

Reply:
Renamed via re-base - how's `remove-recent`?

Reply:
That works for me.